### ISSUE TITLE
Homematic addon broken

### ISSUE THREAD

**Issue by Skyler3189** [home-assistant/hassio-addons#1501]
Hi,
the Homematic addon seems to be broken as neither on a blank installation of hassOS 4.11 on a RPi3 nor on my RPi4 main installation I'm able to access basic pages such as "Zentralenwartung" (screen turns in to "wait" mode with greyed background and stays forever), creating and restoring backups (errors thrown) etc. Independent of interfaces (RF, HmIP) being enabled or not.
The current version of RaspberryMatic runs fine with full functionality on both RPi3 and RPi4 - therefore there seems to be an issue with the integration in hass.io rather than OCCU itself.

Steps to reproduce:
1.) Clean hassOS 4.11 installation (32bit) on RPi3 or RPi4
2.) Install and (default) configure homematic addon 
3.) Open homematic WebUI via ingress and go to "Systemsteuerung" (Control panel) and try to open "Zentralenwartung" or try to create a backup.

Regards,
Skyler

**Comment by pvizeli** [comment]
You can not use the hub functionality, because Home Assistant is the Hub. You can only control and manage you device over ingress.

**Comment by Skyler3189** [comment]
Hi @pvizeli,
thank you for the quick reply! 
In general I want the CCU only for communication with the Homematic devices - all logic is in Hass.io. 

To clarify: 
- Backup and restore procedure in the OCCU addon are offically not supported, correct? Is Device data is saved in the "snapshot" functionality of hassOS? 
- The official migration procedure is "disconnecting" all devices from the actuall HW CCU and add all of them in the OCCU addon manually? 

If both is the case it would be great if this information could be added to the documentation page of the addon as I've tried all weekend long to migrate via backup / restore. It became even more confusing as in RaspberryMatic everything ran like a charm using the same OCCU (afaik) ;)

FYI: I've also tried the direct access of the WebUI / ReGaHss on port 80 (and not just via Ingress). Same error message on backup creation: "An internal error was detected in the service software. The diagnostics are being emailed to the service system administrator (root)."
